### PR TITLE
Fix migration provider

### PR DESCRIPTION
**Pull request by bartv2**
Some more fixes

### CODE REVIEW

**Comment by bartv2** [comment]
@tomneedham

File: appinfo/migrate.php [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <?php
 namespace OCA\Contacts;
 
-class MigrationProvider extends OC_Migration_Provider{
+class MigrationProvider extends \OC_Migration_Provider{
```
Comment (tanghus) [inline appinfo/migrate.php:4]: This was fixed yesterday by https://github.com/owncloud/contacts/pull/310

Comment (bartv2) [inline appinfo/migrate.php:4]: yes i noticed, but this should still merge, so i didn't do a new push ;-)

**Comment by tomneedham** [comment]
:+1:

**Comment by tanghus** [comment]
Great, thanks guys

**Pull request merged**